spec/rails_helper.rb

```diff
@@ -120,7 +120,6 @@ def self.capybara_tmp_path
     config.use_transactional_fixtures = true
 
     DatabaseCleaner.clean_with(:truncation)
-    seed_base_data_for_tests if RSpec.current_example.metadata[:seed_items] != false
```

Comment:
You can actually ignore this line - I remove this entire block in #3490.

Reply:
I will leave it in then